prac_07/guitar.py

```diff
@@ -0,0 +1,17 @@
+class Guitar:
+    """Represent a Guitar object."""
+
+    def __init__(self, name="", year=0, cost=0):
```

Comment:
The default value for cost could be a float (0.0). This won't impact the code functionality, however it shows the reader what type of value to expect for that attribute.